docs/adr/0001-retiring-licence-finder.md

```diff
@@ -0,0 +1,82 @@
+
+Design-wise, the existing tool doesn't match any of the other design patterns on GOV.UK (e.g. smart-answers or finders), nor does it use any design system components.
+
+Finally, the existing tool doesn't contain any extra analytics tracking on the search field or the [sectors] and [activities] list making it difficult to track user behaviour.
+
+## Decision
+
+This Licence Finder application will be retired and replaced with a specialist finder similar to the [Air Accidents Investigation Branch reports] finder.
+
+Specialist Finders find documents with a particular special document schema and are configured in [Specialist Publisher].
+Specialist Finders are an existing pattern on GOV.UK and follow the design system. The specialist finder render already contains analytics tracking for the facets, search box and results clicked.
+
+Another benefit of this approach is that licences will still appear in the GOV.UK site search. Also, the default behaviour of finders is to show all of the available licences to the user when the finder loads, so the user will always see some results.
+
+A new licence document type will be created in Specialist Publisher. This new document type in Specialist Publisher will allow content designers to populate all of the existing fields for licence, as well as tagging the individual licence to sectors and activities via a user interface rather than a rake task. Specialist Publisher already allows publishers in other departments to create content by either being part of a named organisation, or by being assigned a custom permission. This will move the burden of managing licence content to the department that owns them.
+
+Specialist Publisher also generates tagging interfaces for specialist documents from the configuration files, so a new interface will not need to be built.
+
+### Other approaches considered
+
+#### Updating the current tool
+
+This approach was immediately discarded for two main reasons. The current tool is not fit for purpose, and the overhead of needing to update so many licences at once.
+
+#### Creating a "search" finder in search-api
+
+The same benefits of using a specialist finder to display the results also apply to a finder configured in search-api, the finder follows the design system and has analytics out of the box. However the licence format would still need to be migrated to another publishing application so that they can be managed by other government departments as well.
+
+Whitehall was considered as a potential home for the licence format. However migrating a format to Whitehall would be more complex than migrating to Specialist Publisher. Also, this would negatively affect the roadmap for the Publishing team which is doing extensive work to improve Whitehall.
+
+A tagging interface would also need to be created somewhere to tag licences to sectors and activities
```

Comment:
-> 'An admin interface would also need to be created, which would manage the list of sectors and activities and their tagging to licences.'